### DISCUSSION TITLE
SignalR example

### DISCUSSION THREAD

**Discussion by pavelvondrasek** [vercel/next.js#19156] [Help]
Hi,

I use Next on my projects and because I'm doing .NET on backend I tried SignalR. I didn't know how to setup it and opened issue with question: https://github.com/dotnet/aspnetcore/issues/27781 and got answer.

I think we should have SignalR in examples. What do you think? Can I send you PR? It's very simple but maybe it can help. :)

**Comment by marlongerson** [comment]
@pavelvondrasek Do you have an example repo? I am getting "undefined is not a function" when importing @microsoft/signalr package.

**Reply by pavelvondrasek** [reply]
Yes, I've private repo with working SignalR. Should I prepare example for you? :)

**Reply by dleunig** [reply]
That would be awesome! I'm looking for a solution with the same use case.
